iOS/HowtankWidgetSwift.xcframework/ios-arm64_x86_64-simulator/HowtankWidgetSwift.framework/Modules/HowtankWidgetSwift.swiftmodule/x86_64-apple-ios-simulator.swiftinterface

```diff
@@ -251,15 +256,17 @@ public enum ValueCurrency {
     case inactive
     case tabs
     case header
-    public init?(rawValue: Swift.String)
-    public typealias RawValue = Swift.String
-    public var rawValue: Swift.String {
+    public init?(rawValue: Swift.Int)
```

Comment:
Why did you change the type from String to Int?

Reply:
I didn't change anything in the code, It was generated from the latest code so I think the old one is out dated.